src/App.tsx

```diff
@@ -10,31 +11,32 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const onTabSelected = (id: string) => {
+    setSelectedTabId(id);
+  };
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
move to the `Tabs` component
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```